### PR TITLE
fix: keepHeadersOnError shouldn't affect OPTIONS request

### PR DESCRIPTION
**Pull request by dead-horse**
related: #15

### CODE REVIEW

**Comment by fengmk2** [comment]
@dead-horse ci fails

File: test/cors.test.js [hunk 1]
```diff
@@ -342,6 +342,22 @@ describe('cors.test.js', function() {
       .expect(500, done);
     });
 
+    it('should not affect OPTIONS requests', function (done) {
+      var app = new Koa();
```
Comment (fengmk2) [inline test/cors.test.js]: use const instead of var

**Comment by dead-horse** [comment]
fixed

**Comment by fengmk2** [comment]
+1

**Comment by fengmk2** [comment]
2.1.1

**Pull request merged**